apps/src/templates/SchoolDataInputs.jsx

```diff
@@ -33,7 +33,7 @@ export default function SchoolDataInputs({
     detectedCountry === US_COUNTRY_CODE
   );
   const [isOutsideUS, setIsOutsideUS] = useState(
-    detectedCountry && detectedCountry !== US_COUNTRY_CODE
+    (detectedCountry || usIp === false) && detectedCountry !== US_COUNTRY_CODE
```

Comment:
While testing, I found a bug that if `usIp` was passed in as `false`, `isOutsideUS` would still return `false` because it would process `detectedCountry` as `''` (meaning the school name field wouldn't automatically show up). This handles the case now for when `usIp` is `false`, rather than treating `false` and `undefined` as the same for `usIp`.

Reply:
If `usIp` is `false` should `isOutsideUS` still be false?

Reply:
I believe we want the flipped behavior, so if `usIp` is `false` then we know they're outside of the U.S. so `isOutsideUS` should be true (and vice versa). But if `usIp` is `undefined` then we default to `isOutsideUS` being `false`

Reply:
ugh right right, double negatives are hard

Reply:
I think I'm getting tripped up on needing separate behavior for undefined, which is a bit of a code smell

Reply:
okay okay, so it sounds like the original bug is that this line was true when `detectedCountry === ''`, right? Could we change this to just be `!!detectedCountry`?

Reply:
(Sorry, I'm just trying to understand what's going on at this point. A lot of logic here)

Reply:
Super fair! It's just that right now `usIp` isn't really doing anything since we are prioritizing `detectedCountry`, but then if we have no `detectedCountry` then regardless of if `usIp` is `true`, `false`, or `undefined`, it will still cause `isOutsideUS` to be `false`. So we could also remove `usIp` and just always default to U.S. if no `detectedCountry` is present?

Reply:
Yes for sure! The main issue is that `''` is considered `false` when it's checking the first `detectedCountry` on line 36. So if `sessionStorage.getItem(SCHOOL_COUNTRY)` is empty, then `detectedCountry` is only relying on `(usIp ? US_COUNTRY_CODE : '')`. If `usIp` is `true`, then `detectedCountry && detectedCountry !== US_COUNTRY_CODE` returns `false` for `isOutsideUS` (which is what we want). However, if `usIp` is `false` (for sure outside of the U.S.) or `undefined` (unclear if outside U.S.), `(usIp ? US_COUNTRY_CODE : '')` will be evaluated as `''` in both cases. So `usIp` doesn't change the outcome (i.e. if `usIp` is `false`, we aren't treating it like we know that the user is for sure outside of the U.S.): if we have no `detectedCountry`, then `isOutsideUS` is always `false`.

So I think we can either update the logic or remove `usIp`!

Reply:
ooo I think I see. So we should probably be defaulting to assuming outside the US if `usIp` is false, but we're not doing that.

And then there's also the question of how we want to treat `usIp` as undefined, right?

Reply:
I think I'm convinced but would love @hannahbergam to take a look when she's back. I think this PR is blocking you so I'll approve and you can merge, then we can follow up if Hannah has feedback

Reply:
Yep that's right! Sounds good, thank you!